File: CSharp/Kex.cs
using System;

namespace CSharp
{
    class Kex
    {
        public void Execute()
        {
            Random rand = new Random();
            Console.WriteLine("冒険が今始まる！");

            PlayerEx player = new PlayerEx(PlayerEx.INITIAL_HP);
            int kills = 0;
            bool continueGame = true;

            do
            {
                Console.WriteLine($"\n現HP：{player.Hp}");
                Console.Write("奥に進みますか？（１：奥に進む　０．帰る）＞");
                continueGame = Convert.ToBoolean(int.Parse(Console.ReadLine()));

                if (continueGame)
                {
                    int golem_lv = rand.Next(0, 5);
                    Kex_1sub.AttackGolemEx(golem_lv, ref player);
                    ++kills;
                    if (player.Hp == 0)
                        break;
                }

            } while (continueGame);

            Console.WriteLine("リ〇ミト！\n");
            Console.WriteLine($"戦闘回数：{kills}　残りHP：{player.Hp}");
        }
    }
}

File: Python/kex.py
from random import Random
from kex_1sub import attack_golem
from player import PlayerEx


class kex(object):
    @staticmethod
    def execute():
        rand = Random()
        print('冒険が今始まる！')

        player = PlayerEx()
        continue_game = True
        kills = 0
        while continue_game:
            print('現HP：%d' % player.hp)
            print('奥に進みますか？（１：奥に進む　０．帰る）＞', end='')
            continue_game = bool(int(input()))
            if continue_game:
                golem_lv = rand.randint(0, 4)
                attack_golem(golem_lv, player)
                kills += 1
                if player.hp == 0:
                    break
        print('リ〇ミト！\n')
        print('戦闘回数：%d　残りHP：%d' % (kills, player.hp))
